src/open_inwoner/configurations/migrations/0085_create_partial_admin_edit_permissions.py

```diff
@@ -34,7 +38,14 @@ def delete_partial_admin_edit_permissions(apps, _):
         "siteconfig_fieldset_page_texts",
         "siteconfig_fieldset_help_texts",
     ):
-        Permission.objects.get(codename=codename).delete()
+        try:
+            Permission.objects.get(codename=codename).delete()
```

Comment:
```suggestion
            Permission.objects.filter(codename=codename).delete()
```

Would not require the catch. I think it returns the number of deleted rows, you could print that.